Description: Fix spelling and typographic errors in the manual pages.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: no
Last-Update: 2011-03-13

--- a/doc/bsetbg.1
+++ b/doc/bsetbg.1
@@ -112,25 +112,25 @@
 The default is ~/.bsetbg_last_cmd .
 
 .TP
-As mentioned above, \fBbsetbg\fR will function perfectly for the majority of users without having a configuration file. Power users who want more control over \fBbsetbg\fR's behavior should run \fBbsetbg -g\fR and use the output to create a \fI~/.bsetbgrc\fR which may then be tweaked by hand.
+As mentioned above, \fBbsetbg\fR will function perfectly for the majority of users without having a configuration file. Power users who want more control over \fBbsetbg\fR's behavior should run \fBbsetbg \-g\fR and use the output to create a \fI~/.bsetbgrc\fR which may then be tweaked by hand.
 
 .SH EXAMPLES
 In this example, bsetbg will set the image in centered mode:
 
-    bsetbg -center foo.png
+    bsetbg \-center foo.png
 
 An example of the \fB-exec\fR argument:
 
-    bsetbg -exec xv -root -quit -rmode 5 -rbg rgb:2/2/2 \\
-	-center foo.png
+    bsetbg \-exec xv \-root \-quit \-rmode 5 \-rbg rgb:2/2/2 \\
+	\-center foo.png
 
 An example in which bsetbg creates a configuration file using xv and qiv:
 
-	bsetbg -g xv qiv > ~/.bsetbgrc
+	bsetbg \-g xv qiv > ~/.bsetbgrc
 
 An example of the use of the \fB-app\fR argument:
 
-    bsetbg  -app qiv "-o rgb:d6/c5/a2 -x" -c foo.png
+    bsetbg \-app qiv "\-o rgb:d6/c5/a2 \-x" \-c foo.png
 
 .SH AUTHOR
 The author of
--- a/doc/blackbox.1.in
+++ b/doc/blackbox.1.in
@@ -251,7 +251,7 @@
 mentioned in the Description. Dockapps 
 automatically run in the slit in most cases, but
 may require a special command switch.  
-Often, -w is used for "withdrawn" into the slit.
+Often, \-w is used for "withdrawn" into the slit.
 
 gkrellm is a very useful and modern dockapp that
 gives the user near real time information on 
@@ -280,8 +280,8 @@
 .EX 8
 .B i.e.
 #!/bin/sh
-speyes -w & sleep 2
-gkrellm -w & sleep 2
+speyes \-w & sleep 2
+gkrellm \-w & sleep 2
 .EE 
 .\"
 .\" ***** THE TOOLBAR *****
@@ -393,7 +393,7 @@
 workspaces.
 .TP
 .B Kill Client
-This kills the client program with -SIGKILL (-9)
+This kills the client program with \-SIGKILL (\-9)
 Only use this as a last resort.
 .TP
 .B Close
@@ -704,7 +704,7 @@
 .B Here is a working example of a menu file:
 .\" ----- menu example -----
 [begin] (MenuName)
-   [exec] (xterm) {xterm -ls -bg black -fg green}
+   [exec] (xterm) {xterm \-ls \-bg black \-fg green}
    [submenu] (X utilities)
       [exec] (xcalc) {xcalc}
    [end]
@@ -847,7 +847,7 @@
 .TP 3
 .BI "session.imageDither" "  [True|False]" 
 This setting is only used when running in low 
-color modes. Image Dithering helps to show an
+color modes. Image Dithering helps one to show an
 image properly even if there are not enough
 colors available in the system.
 .EX 
